src/main/java/org/jitsi/videobridge/cc/vp8/VP8Frame.java

```diff
@@ -27,16 +27,11 @@
  * time of the creation of this instance.
  *
  * Instances of this class are *NOT* thread safe. While most internal state of
- * this class instances is final, the maxSequenceNumber and the
- * endingSequenceNumberIsKnown fields are not and it is therefore necessary
- * to synchronize reading/writing to those fields on this instance.
- *
- * Consider, for example, creating a new frame projection (that happens in the
- * translator thread and reads the maxSequenceNumber field) while packets are
- * being piggy-backed (that happens in the egress thread and could potentially
- * write the maxSequenceNumber field).
+ * this class instances is final, the sequence number ranges, haveStart/haveEnd,
+ * and isKeyframe.
```

Comment:
I think this sentence is missing something?

Reply:
Fixed.